test/cashu/secret.test.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, expect, test } from 'bun:test';
+import { parseSecret } from '../../lib/cashu/secret';
+
+describe('parseSecret', () => {
```

Comment:
I would also add one test for successful parse of nut 10 secret and one for failure due to invalid nut 10 secret

Reply:
added a bunch more tests